micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspect.groovy

```diff
@@ -56,5 +66,23 @@ class MetricsAspect {
                 trimLeadingCharacter(str, '.' as Character), '.' as Character)
     }
 
+    public static String sanatizeIpAddressIfPresent(String metricName) {
```

Comment:
InetAddressValidator can be used here. There are 2 mothods:
- isValidInet4Address(String inet4Address)
- isValidInet6Address(String inet6Address)

What do you think?